data/gegno/gegno intro missions.txt

```diff
@@ -406,48 +406,23 @@ mission "Giaru Gegno: Revisit"
 				decline
 
 
+# Gegno Superheavy Missions
 
-# Gegno Superheavy
-
-mission "Stationed Augen"
+mission "Vi Augen"
 	invisible
 	landing
-	npc save
-		government "Gegno Vi"
-		personality heroic vindictive staying
-		system "Kiluit"
-		ship "Augen" "Vytii Yrrlausei"
-	to fail
-		has "event: Vesvii war"
-	to complete
-		never
-
-mission "Augen Destruction"
-	landing
-	invisible
-	to offer
-		has "Stationed Augen: failed"
-		not "event: Vesvii war"
 	on offer
-		set "dead augen"
-		fail
-
-mission "Active Augen"
-	invisible
-	to offer
-		has "event: Vesvii war"
-		has "Stationed Augen: failed"
-		not "dead augen"
+		fail "Stationed Augen"
```

Comment:
Never been good with NPC, so this might be my own lack of knowledge - but `Stationed Augen` is not mentioned in the file outside of this one spot. Is that correct?

Reply:
It's to fail the existing mission. In the current build, that's the name of the NPC mission, and in this one, it fails it if the player has it to spawn the newer one. This was also done with https://github.com/endless-sky/endless-sky/pull/8594.